superbench/benchmarks/micro_benchmarks/dist_inference_cpp.py

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
I think all should be very similar as previous one

Reply:
Merged.